paasta_tools/kubernetes/bin/paasta_secrets_sync.py

```diff
@@ -266,12 +268,17 @@ def sync_boto_secrets(
                     )
         if not secret_data:
             continue
+        # In order to prevent slamming the k8s API, add some artificial delay here
+        time.sleep(0.3)
         app_name = get_kubernetes_app_name(service, instance)
-        secret = f"paasta-boto-key-{app_name}"
-        hashable_data = "".join([secret_data.get(key, "") for key in secret_data])
+        secret = f"{namespace}-boto-key-{app_name}"
```

Comment:
i think we actually wanna keep this as `paasta` since some namespaces can be extremely large (e.g., the nrtsearch namespace)